docs/source/troubleshooting/troubleshooting-getting-started.adoc

```diff
@@ -138,3 +138,14 @@ Downloading https://github.com/minishift/minishift/releases/download/v1.2.0/mini
  65 B / 65 B [===================================================================================================================================================] 100.00% 0s
 Update failed: open /usr/bin/.minishift.new: permission denied
 ----
+
+[[safari-openshift-web-console-issue]]
+== Safari browser issue with OpenShift web console
+
+OpenShift web console URL i.e. `minishift console` does not work on older versions of Safari web browser e.g. version 10.1.2 (12603.3.8) and its gives following error:
```

Comment:
Typo s/its/it ..

Reply:
Ins't "Safari browser issue with OpenShift web console" not a little TOO generic? 
s/of Safari/of the Safari
Also, eg, can not be part of the same sentence here.